Jenkinsfile

```diff
@@ -390,12 +392,10 @@ def cleanDir(path){
 }
 
 def dotnet(Closure body){
-  def home = "/tmp"
-  def dotnetRoot = "/${home}/.dotnet"
-  def path = "/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/${home}/bin:${dotnetRoot}:${dotnetRoot}/bin:${dotnetRoot}/tools"
-  docker.image('mcr.microsoft.com/dotnet/core/sdk:2.1.505').inside("-e HOME='${home}' -e PATH='${path}'"){
-    sh 'chmod +x .ci/linux/tools.sh'
-    sh '.ci/linux/tools.sh'
+  def home = '/tmp'
+  def dotnetRoot = '/usr/share/dotnet'
```

Comment:
This is the location of the dotNet installation, although worth to check whether it's required when using `mcr.microsoft.com/dotnet/core/sdk:3.1.100`